tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -643,6 +772,51 @@ class MklConvOp : public OpKernel {
 
       // Check whether src and filter need to be reordered
       Tinput* src_data = nullptr;
+#ifdef ENABLE_MKLDNN_V1
+      if (src_md != conv_fwd_pd->src_desc()) {
+        // Reorder src
+        src.SetUsrMem(src_md, &src_tensor);
+        src.CheckReorderToOpMem(conv_fwd_pd->src_desc(), cpu_engine_);
+        src_data = static_cast<Tinput*>(src.GetOpMem().get_data_handle());
+      } else {
+        src_data = static_cast<Tinput*>(
+            const_cast<Tinput*>(src_tensor.flat<Tinput>().data()));
+      }
+
+      Tfilter* filter_data = nullptr;
+      if (filter_md != conv_fwd_pd->weights_desc()) {
```

Comment:
For this line, please feel free to just wrap `#ifdef` around just this line.

Reply:
Done.